src/test/incremental/hashes/unary_and_binary_exprs.rs

```diff
@@ -454,7 +469,9 @@ pub fn lvalue() -> i32 {
 #[rustc_metadata_dirty(cfg="cfail2")]
 #[rustc_metadata_clean(cfg="cfail3")]
 pub fn lvalue() -> i32 {
-    let y = 10;
+    let mut x = 10;
+    let mut y = 11;
+    x = 9;
     y
```

Comment:
We want to change the left-hand side of the assignment here, not the return value of the function.